src/gridfs-stream/download.ts

```diff
@@ -262,7 +314,7 @@ function init(self: any) {
     findOneOptions.skip = self.s.options.skip;
   }
 
-  self.s.files.findOne(self.s.filter, findOneOptions, (error?: any, doc?: any) => {
+  self.s.files.findOne(self.s.filter, findOneOptions, (error?: AnyError, doc?: GridFsFile) => {
```

Comment:
It would be nice to drop the annotations from the callback.
Maybe we should go in and fix the Collections type to have the deprecate functions as class properties.
Or we should update the Collection interface to be better typed.

Reply:
I merged these methods into the main class yesterday, you should be able to use it without annotations here now